sigma9.css

```diff
@@ -112,8 +114,13 @@ a.newpage {
 
 body {
 	background-color: #fff;
-	font-size: .8em;
+	font-size: .95em;
 	color: #333;
+	font-family: "Inter", BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
```

Comment:
Out of curiosity, why are we not using the Variable Font version of Inter (specifically `'Inter var experimental'`)? It's pretty widely supported at this point and I don't see much reason for not using it. It's also what BHL uses but it is self-hosted instead of importing from `rsms.me`.

Reply:
Agreeing with this.